src/components/survey/owner.tsx

```diff
@@ -197,14 +197,31 @@ const Owner: React.FC<Props> = ({ surveyId }) => {
             onOwnerInfoContactPersonPropChange
           )
         }
-        {
-          renderWithDebounceTextField(
-            "email",
-            strings.survey.owner.email,
-            ownerInformation?.contactPerson?.email || "",
-            onOwnerInfoContactPersonPropChange
-          )
-        }
+        <Stack
+          direction="row"
+          alignItems="center"
+          sx={{
+            display: "flex", justifyContent: "space-between", gap: 1
+          }}
+        >
+          {
+            renderWithDebounceTextField(
+              "email",
+              strings.survey.owner.email,
+              ownerInformation?.contactPerson?.email || "",
+              onOwnerInfoContactPersonPropChange
+            )
+          }
+          {/* added a non-functional button, feature added later */}
+          <Button
+            variant="contained"
+            color="primary"
+            sx={{ width: "200px", height: "50px" }}
+          >
+            {strings.survey.owner.sendLink}
+
+          </Button>
```

Comment:
Just a note for future development: I would suggest that clicking the send-button opens a modal/dialog where user can see the address that the link is been sent and also some kind of description what is happening. There would also be the "Cancel" and "Send" button that actually triggers the sending function.